Source/Scene/Scene.js

```diff
@@ -868,11 +868,9 @@ define([
                 break;
             }
 
-            if (command.pass === Pass.OPAQUE || command instanceof ClearCommand) {
-                frustumCommands.opaqueCommands[frustumCommands.opaqueIndex++] = command;
-            } else if (command.pass === Pass.TRANSLUCENT){
-                frustumCommands.translucentCommands[frustumCommands.translucentIndex++] = command;
-            }
+            var pass = command instanceof ClearCommand ? Pass.OPAQUE : command.pass;
+            var index = frustumCommands.indices[pass]++;
+            frustumCommands.commands[pass][index] = command;
```

Comment:
I know this was already here, but are we implicitly growing the array?  Or do we resize it elsewhere?